modules/sdk/project-templates/project-templates-portletmvc4spring-jsp-portlet/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/liferay-portlet.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
```

Comment:
@gamerson The PortletMVC4Spring ones can stay at 7.1 since they work for 7.1 and 7.2. It is a least common denominator. However if the pattern elsewhere is to make it conditional I can do that.